docs/en/dev_setup/dev_env_mac.md

```diff
@@ -4,109 +4,78 @@ The following instructions set up a PX4 development environment for macOS.
    :::
 
-1. Enforce Python 3 by appending the following lines to `~/.zshenv`
-
-   ```sh
-   # Point pip3 to macOS system python 3 pip
-   alias pip3=/usr/bin/pip3
-   ```
+3. Enforce Python 3 by appending the following lines to `~/.bashrc`(or `~/.zshrc` for zsh):
 
-### Common Tools
+   Ensure Python 3 is used by default, as some PX4 scripts require the `python3` and `pip3` executables to be available in your system `PATH`.
 
-To setup the environment to be able to build for Pixhawk/NuttX hardware (and install the common tools for using simulators):
+   ::: tip
+   If you need help installing an updated version of Python 3, we recommend the [pyenv project](https://github.com/pyenv/pyenv?tab=readme-ov-file), which gives you utmost flexibility by allowing you to set a global and local Python version at the per-directory level.
+   :::
 
-1. Install Homebrew by following these [installation instructions](https://brew.sh).
-1. Run these commands in your shell to install the common tools:
+4. Point pip3 to macOS system python 3 pip
 
    ```sh
-   brew tap PX4/px4
-   brew install px4-dev
+   alias pip3=/usr/bin/pip3
    ```
 
-1. Install the required Python packages:
+### Install Development Tools
+
+1. Install the PX4 development toolchain and simulation environment:
 
    ```sh
-   # install required packages using pip3
-   python3 -m pip install --user pyserial empty toml numpy pandas jinja2 pyyaml pyros-genmsg packaging kconfiglib future jsonschema
-   # if this fails with a permissions error, your Python install is in a system path - use this command instead:
-   sudo -H python3 -m pip install --user pyserial empty toml numpy pandas jinja2 pyyaml pyros-genmsg packaging kconfiglib future jsonschema
+   brew tap PX4/px4
+   brew install px4-dev px4-sim
    ```
 
-## Gazebo Classic Simulation
+2. ### Download PX4 Source Code
 
-To setup the environment for [Gazebo Classic](../sim_gazebo_classic/index.md) simulation:
+```sh
+git clone https://github.com/PX4/PX4-Autopilot.git --recursive
+```
 
-1. Run the following commands in your shell:
+4. Prerequisites setup
 
-   ```sh
-   brew unlink tbb
-   sed -i.bak '/disable! date:/s/^/  /; /disable! date:/s/./#/3' $(brew --prefix)/Library/Taps/homebrew/homebrew-core/Formula/tbb@2020.rb
-   brew install tbb@2020
-   brew link tbb@2020
-   ```
-
-   ::: info
-   September 2021: The commands above are a workaround to this bug: [PX4-Autopilot#17644](https://github.com/PX4/PX4-Autopilot/issues/17644).
-   They can be removed once it is fixed (along with this note).
-   :::
-
-1. To install SITL simulation with Gazebo Classic:
+   Use the [`macos.sh` setup script](https://github.com/PX4/PX4-Autopilot/blob/main/Tools/setup/macos.sh) to automatically install all development dependencies:
 
    ```sh
-   brew install --cask temurin
-   brew install --cask xquartz
-   brew install px4-sim-gazebo
+   git clone https://github.com/PX4/PX4-Autopilot.git --recursive   cd PX4-Autopilot
+   ./Tools/setup/macos.sh
```

Comment:
@farhangnaderi there are a lot of very basic errors here. Did you build the docs and look at them?

```suggestion
   cd PX4-Autopilot
   ./Tools/setup/macos.sh
```